docs/conftest.py

```diff
@@ -112,6 +112,71 @@ def existing_bucket_name(s3_client):
     delete_bucket_and_wait(s3_client, bucket_name)
 
 @pytest.fixture
+def empty_bucket_name(s3_client):
+    # Generate a unique name for the bucket to simulate an empty bucket
+    bucket_name = generate_unique_bucket_name(base_name="empty-bucket")
+
+    create_bucket_and_wait(s3_client, bucket_name)
+    
+    # Yield the empty bucket name to the test
+    yield bucket_name
+
+    objects = s3_client.list_objects(Bucket=bucket_name).get("Contents", [])
+
+    if objects:
+        for obj in objects:
+            s3_client.delete_object(Bucket=bucket_name, Key=obj["Key"])
+    
+    # Teardown: delete the bucket after the test
+    delete_bucket_and_wait(s3_client, bucket_name)
+
+@pytest.fixture
+def create_big_file_with_two_parts():
+    object_key = "large_file.txt"
+    with open(object_key, "w") as f:
+        f.write("A" * 10 * 1024 * 1024 * 5)
+
+    # Dividindo o arquivo em 2 partes
+    total_size = 10 * 1024 * 1024 * 5  # Tamanho total em bytes (50 MB)
+    part_sizes = [total_size // 2] * 2  # Cada parte terá metade do tamanho
+
+    # Se o tamanho total não for divisível por 2, ajusta a última parte
+    part_sizes[-1] += total_size % 2
+
+    part_files = []
+    with open(object_key, "r") as f:
+        for i, size in enumerate(part_sizes):
+            part_path = f"{object_key.split('.')[0]}_part_{i+1}.txt"
+            with open(part_path, "w") as part_file:
+                part_file.write(f.read(size))
+            part_files.append(part_path)
+    
+    yield object_key, part_files
+
+    os.remove(object_key)
+
+    for i in range(2):
+        os.remove(f"{object_key.split('.')[0]}_part_{i+1}.txt")
+
+
+@pytest.fixture
+def bucket_with_one_object_and_cold_storage_class(s3_client):
```

Comment:
content = b"Sample content for testing presigned URLs." é meio confuso nesse contexto de storage class.